src/Admin/CitizenProjectAdmin.php

```diff
@@ -361,6 +361,22 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
             ->add('featured', null, [
                 'label' => 'Coup de coeur',
             ])
+            ->add('turnkeyProject', CallbackFilter::class, [
+                'label' => 'Type de projet',
+                'show_filter' => true,
+                'field_type' => ChoiceType::class,
+                'field_options' => [
+                    'choices' => [
+                        'Projets simples' => CitizenProject::SIMPLE_TYPE,
+                        'Projets clés en main' => CitizenProject::TURKNEY_TYPE,
+                    ],
+                ],
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    $qb->andWhere(\sprintf('%s.turnkeyProject is %s', $alias, CitizenProject::TURKNEY_TYPE === $value['value'] ? 'not null' : 'null'));
```

Comment:
Add a backslash on the `sprintf` function is not needed.

We use the `compiler_optimized` option in PHP CS Fixer. ( https://github.com/EnMarche/en-marche.fr/blob/master/.php_cs.dist#L23 )

You can see all functions where it is needed here : https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Fixer/FunctionNotation/NativeFunctionInvocationFixer.php#L350